webCommander/centralHub/spotify/spotifyAPI.py

```diff
@@ -59,7 +59,31 @@ def get_spotify_artist(artist_name):
     response = requests.get(url, headers=spotify_authenticate())
     return response.text
 
+def search_spotify_artist_by_name(artist_name):
+    url = SPOTIFY_BASE_URL + SPOTIFY_SEARCH
+
+    params = {
+        "q": artist_name,
+        "type": "artist"
+    }
+
+    response = requests.get(url, headers=spotify_authenticate(), params=params)
+    artists_results = json.loads(response.text)
+
+    for artist in artists_results['artists']['items']:
+        if artist['name'] == artist_name:
+            name = artist['name']
+            popularity = artist['popularity']
+            if len(artist['images']) > 0:
+                album_image = artist['images'][0]['url']
+            else:
+                album_image = "N/A"
+    return (name,album_image,popularity)
```

Comment:
could return grouped object here, for example a namedtuple, see also https://www.youtube.com/watch?v=fJmZAsulP7g